### PR TITLE
Fix creation test users on Checkout Marketplace

### PR DESCRIPTION
**Pull request by adfonseca**
## Description
Standardization of creating test users on Checkout Marketplace.

### Checklist:
Before sending your contribution, please specify the following: 

- [x ] This request modifies code snippets. 
- [x ] The contribution aligns with the information stated in the repository wiki.

- [ ] Contribution incorporates an article not included until now, which must be added to indexes as a new section. 
- [ ] Contribution includes new features. 
- [ ] New features were communicated in changelog.
- [ ] Requires translations.

### CODE REVIEW

File: guides/marketplace/web-checkout/testing-marketplace.en.md [hunk 1]
```diff
@@ -25,23 +25,19 @@ To simulate the entire payment process, you must create 3 test users: **marketpl
...
-# Get access token
-AT=`curl -s -X POST -H 'content-type: application/x-www-form-urlencoded' 'https://api.mercadopago.com/oauth/token'
--d 'grant_type=client_credentials'
--d 'client_id=CLIENT_ID'
--d 'client_secret=CLIENT_SECRET' | grep -o '"access_token":"[^"]*"' | sed -n 's/.*"access_token":"\(.*\)"/\1/p'`
-
 curl -X POST \
 -H "Content-Type: application/json" \
-"https://api.mercadopago.com/users/test_user?access_token=$AT" \
+"https://api.mercadopago.com/users/test_user?access_token=PROD_ACCESS_TOKEN" \
```
Comment (lamaralmp) [inline guides/marketplace/web-checkout/testing-marketplace.en.md:33]: @adfonseca poderia alterar para ACCESS_TOKEN conforme falamos no Slack?

File: guides/marketplace/web-checkout/testing-marketplace.es.md [hunk 2]
```diff
@@ -24,23 +24,19 @@ Para simular el proceso de pago de punta a punta debes crear 3 usuarios de prueb
...
-# Get access token
-AT=`curl -s -X POST -H 'content-type: application/x-www-form-urlencoded' 'https://api.mercadopago.com/oauth/token'
--d 'grant_type=client_credentials'
--d 'client_id=CLIENT_ID'
--d 'client_secret=CLIENT_SECRET' | grep -o '"access_token":"[^"]*"' | sed -n 's/.*"access_token":"\(.*\)"/\1/p'`
-
 curl -X POST \
 -H "Content-Type: application/json" \
-"https://api.mercadopago.com/users/test_user?access_token=$AT" \
+"https://api.mercadopago.com/users/test_user?access_token=PROD_ACCESS_TOKEN" \
```
Comment (lamaralmp) [inline guides/marketplace/web-checkout/testing-marketplace.es.md:32]: @adfonseca poderia alterar para ACCESS_TOKEN conforme falamos no Slack?

File: guides/marketplace/web-checkout/testing-marketplace.pt.md [hunk 3]
```diff
@@ -26,23 +26,18 @@ Para simular o processo de pagamento do começo ao fim, é necessário criar 3 u
...
-# Get access token
-AT=`curl -s -X POST -H 'content-type: application/x-www-form-urlencoded' 'https://api.mercadopago.com/oauth/token'
--d 'grant_type=client_credentials'
--d 'client_id=CLIENT_ID'
--d 'client_secret=CLIENT_SECRET' | grep -o '"access_token":"[^"]*"' | sed -n 's/.*"access_token":"\(.*\)"/\1/p'`
-
 curl -X POST \
 -H "Content-Type: application/json" \
-"https://api.mercadopago.com/users/test_user?access_token=$AT" \
+"https://api.mercadopago.com/users/test_user?access_token=PROD_ACCESS_TOKEN" \
```
Comment (lamaralmp) [inline guides/marketplace/web-checkout/testing-marketplace.pt.md:34]: @adfonseca poderia alterar para ACCESS_TOKEN conforme falamos no Slack?

File: guides/marketplace/web-checkout/testing-marketplace.en.md (hunk 1 above)
Comment (adfonseca) [inline guides/marketplace/web-checkout/testing-marketplace.en.md:33]:
```suggestion
"https://api.mercadopago.com/users/test_user?access_token=ACCESS_TOKEN" \
```

File: guides/marketplace/web-checkout/testing-marketplace.es.md (hunk 2 above)
Comment (adfonseca) [inline guides/marketplace/web-checkout/testing-marketplace.es.md:32]:
```suggestion
"https://api.mercadopago.com/users/test_user?access_token=ACCESS_TOKEN" \
```

File: guides/marketplace/web-checkout/testing-marketplace.pt.md (hunk 3 above)
Comment (adfonseca) [inline guides/marketplace/web-checkout/testing-marketplace.pt.md:34]:
```suggestion
"https://api.mercadopago.com/users/test_user?access_token=ACCESS_TOKEN" \
```

**Pull request merged**